### PR TITLE
feat: server logging, avoids duplicate user registration

### PR DESCRIPTION
**Pull request by StoneSteel27**

### CODE REVIEW

File: server/server/managers.py [hunk 1]
```diff
@@ -41,6 +41,14 @@ def get_user(self, user_id: str = "") -> Dict:
         else:
             return self.users
 
+    def does_username_exist(self, username: str) -> bool:
+        """Checks for user with same username"""
+        users = self.get_user()
+        for i in users.values():
+            if i["username"] == username:
+                return True
+        return False
```
Comment (p0lygun) [inline server/server/managers.py:50]:
```suggestion
        return any(user["username"]==username for user in users.value())
```

**Review by p0lygun — CHANGES_REQUESTED** [review]
looks good just one minor change

**Comment by p0lygun** [comment]
and pls also return username when u add a user for the first time

**Comment by StoneSteel27** [comment]
You are the one who's sending the the username to me, you should have it right

**Comment by StoneSteel27** [comment]
You are the one who's sending the the username to me, and after login, i do send the username

**Comment by p0lygun** [comment]
no the client adds user via user_id

**Review by p0lygun — APPROVED** [review]

**Pull request merged**